test/test_heuristics.rb

```diff
@@ -494,6 +494,13 @@ def test_sc_supercollider_scala_by_heuristics
     })
   end
 
+  def test_sol_by_heuristics
+    assert_heuristics({
+      "Solidity" => Dir.glob("#{fixtures_path}/Solidity/{legacy,modern}*"),
+      nil        => Dir.glob("#{fixtures_path}/Solidity/ignored*")
```

Comment:
If those files are not Solidity, then they should not be in the Solidity directory.

Reply:
The `fixtures_path` only contains files used by Linguist's test suite. They're ignored by the classifier (and the rest of Linguists runtime, for that matter).

Reply:
Sure, but it doesn't change the fact they don't belong in `fixtures/Solidity`. I think we have `Data/` for fixture files we're unable to classify. I'd also like to see examples of other languages using `.sol`.

Reply:
I'd prefer we keep all Solidity-related fixtures somewhere obvious. I'll split `fixtures/Solidity` into two directories:

~~~
./test/fixtures/Solidity/match/
./test/fixtures/Solidity/ignore/
~~~